framework/doc/content/syntax/Functors/index.md

```diff
@@ -17,6 +18,11 @@ but for `Functors`, the space arguments can be an element, a point in an element
 a face of an element, and so on. The time arguments represent the state of the functor : current, previous value (whether in time or iteration),
 or value before that previous one.
 
+!alert warning title=Using post-processors
+Note that when using post-processors, instead of being evaluated on-the-fly at
+a point in space and time, the most recently computed value is used, in accordance
+with the `execute_on` for the post-processor.
+
```

Comment:
add a note to the PP documentation saying it inherits the functor interface. Functions and variables doc should have a note already